### ISSUE TITLE
Huge link description

### ISSUE THREAD

**Issue by Nono-m0le** [RocketChat/Rocket.Chat.Electron#2134]
**Describe the bug**
Posting a specific link give a huge description of this link

What operating system and which version : Windows 10 x64
Which version of Rocket.Chat (Server) 3.17.2
Which version of Rocket.Chat.Electron (Electron/Desktop)? 3.4.0

**To Reproduce**
paste the following link : https://synzack.github.io/Blinding-EDR-On-Windows/
get a full size of link description

**Expected behavior**
a few lines of description only

**Screenshots**
![image](https://user-images.githubusercontent.com/3726435/131313954-b47fd93f-a29a-4e28-a1ec-4075d28513d4.png)

**Comment by ankar84** [comment]
I confirm that behavior, but it's not an Electron bug, it's Rocket.Chat bug, because it reproducible in Web client.
@debdutdeb @yashovardhan please transfer that issue to https://github.com/RocketChat/Rocket.Chat/issues
And @Nono-m0le please fill all necessary bug report field after that.
Thanks!

**Comment by Nono-m0le** [comment]
> And @Nono-m0le please fill all necessary bug report field after that.
 you mean, once @debdutdeb or @yashovardhan have transfered the ticket, IF any field are missing ?

**Comment by ankar84** [comment]
> > And @Nono-m0le please fill all necessary bug report field after that.
> 
> you mean, once @debdutdeb or @yashovardhan have transfered the ticket, IF any field are missing ?

Exactly, there is another bug report template with additional fields to fill.
